enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,243 @@
+---
+title: monitoring-windows-nodes
+authors:
+  - "@VaishnaviHire"
+  - "@PratikMahajan"
+reviewers:
+  - "@@openshift/openshift-team-windows-containers"
+  - "@simonpasquier"
+  - "@spadgett"
+approvers:
+  - "@aravindhp"
+  - "@simonpasquier"
+creation-date: 2021-02-08
+last-updated: 2021-03-04
+status: implementable
+---
+
+# Monitoring Windows Nodes
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to enable performance monitoring on Windows
+nodes created by Windows Machine Config Operator(WMCO) in OpenShift cluster.
+
+## Motivation
+
+Monitoring is critical to identify issues with nodes, containers running on the
+nodes. The main motivation behind this enhancement is to enable monitoring on
+the Windows nodes.
+
+### Goals
+
+As part of this enhancement, we plan to do the following:
+* Run [windows_exporter](https://github.com/prometheus-community/windows_exporter)
+  as a service on Windows nodes
+* Upgrade the windows_exporter on the Windows Nodes
+* Leverage cluster-monitoring operator that sets up Prometheus, Alertmanager
+  and other components
+
+### Non-Goals
+
+As part of this enhancement, we do not plan to do the following:
+* Integrating windows_exporter with cluster monitoring operator
+* Ship Grafana dashboards for Windows Nodes
```

Comment:
It looks like we do plan to do this now? (Using console Monitoring -> Dashboards.)